truenas/webui
# NAS-116067 / 23.10 / Ask redirect to ACL editor

**Testing**

In the **Shares > Windows SMB shares** block,

1. Click **Add** fill the basic fields, and click **Advanced settings** button.

2. Click **Submit** button

- **Expected result:** 

   - Pop up a screen asking user if I want to **Configure the ACL**
